src/test/test-helper.c

```diff
@@ -27,3 +28,8 @@ int enter_cgroup_subroot(void) {
 
         return cg_attach_everywhere(supported, cgroup_subroot, 0, NULL, NULL);
 }
+
+/* https://docs.travis-ci.com/user/environment-variables#default-environment-variables */
+bool is_run_on_travis_ci(void) {
+        return streq_ptr(getenv("TRAVIS"), "true");
```

Comment:
Maybe use `getenv_bool` for this?

Reply:
Generally I have no problem with `getenv_bool`, but in this case it would just make it easier to skip the test and I'd like to avoid that. After all, it's a kludge which should be hard to use and which should look as ugly as possible :-)

Reply:
OK.